dsl/src/main/scala/org/http4s/dsl/package.scala

```diff
@@ -128,6 +129,7 @@ package object dsl extends Http4s {
   implicit class PreconditionRequiredSyntax(val status: PreconditionRequired.type) extends AnyVal with EntityResponseGenerator
   implicit class TooManyRequestsSyntax(val status: TooManyRequests.type) extends AnyVal with EntityResponseGenerator
   implicit class RequestHeaderFieldsTooLargeSyntax(val status: RequestHeaderFieldsTooLarge.type) extends AnyVal with EntityResponseGenerator
+  implicit class UnavailableForLegalReasons(val status: UnavailableForLegalReasons.type) extends AnyVal with EntityResponseGenerator
```

Comment:
Looks like you need to add a 'Syntax' suffix here or we have a symbol name issue.